expo-migration-plan.md

```
@@ -33,42 +36,63 @@
+| -------------------------------- | -------------------- | ----------------- |
+| `@capacitor-community/sqlite`    | `expo-sqlite`        | SQL文の互換性確認 |
+| `@capacitor/preferences`         | `expo-secure-store`  | API変更対応       |
+| `@capacitor/local-notifications` | `expo-notifications` | 通知設定の再実装  |
+
+#### 2.3 開発環境設定
+
 ```bash
 npx create-expo-app --template
 ```
 
 #### 2.2 依存関係の移行
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)